### ISSUE TITLE
Gutenboarding launch flow: improve a11y

### ISSUE THREAD

**Issue by razvanpapadopol** [Automattic/wp-calypso#44697]
There are some basic a11y features we should cover for the redesigned launch flow.
- User should be able to navigate between steps, inside each step and launch using keyboard.
- Any focusable element should have a focus state when user tabs to it. Right now sidebar buttons don't have any focus state.
- [...]
